### PR TITLE
Go12 jar support

### PR DESCRIPTION
**Pull request by sciabarracom**
- Support for Go 1.12.4
- Support for jar not expanded for Java when set OW_SAVE_JAR
- You can initalize multiple times when debugging
- Removed gogradle plugin, now building directly with go

### CODE REVIEW

**Comment by csantanapr** [comment]
+1 building directly with Go !!!

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,7 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
+github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
+github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/testify v1.3.0 h1:TivCn/peBQ7UY8ooIcPgZFpTNSz0Q2U6UrFlUfqbe0Q=
+github.com/stretchr/testify v1.3.0/go.mod h1:M5WIy9Dh21IEIfnGCwXGc5bZfKNJtfHm1UVUgZn+9EI=
```
Comment (csantanapr) [inline go.sum]: Do we need this file in the repo or should be ignored?

Comment (neuronix-cloud) [reply go.sum]: yes it is part of go modules support

Comment (sciabarracom) [reply go.sum]: yes it is used by go modules to locate the right version

Comment (csantanapr) [reply go.sum]: Good to know I was not 💯% sure

**Pull request merged**